app/components/RegionDropdown/RegionDropdown.module.scss

```diff
@@ -1,53 +1,45 @@
-@import "../../styles/typography";
+@import "../../styles/colors";
 
 .dropdown-container {
   position: relative;
   cursor: pointer;
-  min-width: 15ch;
-  max-width: 30ch;
-  &:focus-within {
-    .left-arrow {
-      transform: rotate(-45deg);
-    }
-    .right-arrow {
-      transform: rotate(45deg);
-    }
-  }
+  width: 16.25rem;
+  color: var(--text-positive);
+  // Animation is currently off
+
+  // &:focus-within {
+  //   .caret-left{
+  //     transform: rotate(-45deg);
+  //   }
+  //   .caret-right {
+  //     transform: rotate(45deg);
+  //   }
+  // }
 }
+
 .dropdown {
+  font-size: 1.25rem;
   height: 4.25rem;
   appearance: none;
   background-color: transparent;
-  color: var(--text-positive);
   border: none;
-  @include h2;
   font-weight: 500;
   width: 100%;
   padding: 1rem 1.5rem 1rem 2rem;
   border-radius: 8px;
   cursor: pointer;
+  color: inherit;
 
   &:focus {
     background: rgba(255, 255, 255, 0.05);
```

Comment:
Can we drop default outline etc here?

Reply:
Yeah we can, but do we have an alternative style, especially for light version

Reply:
cc @finnclark

Reply:
Could do a simple "ring" using tailwind for now, use ring-current so it matches text?

https://tailwindcss.com/docs/ring-color

Reply:
Should we avoid adding tailwind stuff if we're moving away from it @andycarrell ?

Reply:
As something to unblock this PR and be ready to ship I think it's fine? Also, we can then copy that styling if we're happy with it